src/emulate.c

```diff
@@ -589,7 +581,7 @@ static bool do_fuse5(riscv_t *rv,
 }
 
 /* clang-format off */
-static const void *dispatch_table[] = {
+const void *dispatch_table[] = {
```

Comment:
Ditto. The dispatch table should be hidden.

Reply:
> Ditto. The dispatch table should be hidden.

How about following the approach used for the `ebreak_handler`? We could register the `trap_handler` with the I/O interface during RISC-V core initialization. This way, the dispatch table can remain hidden.

Reply:
> How about following the approach used for the `ebreak_handler`? We could register the `trap_handler` with the I/O interface during RISC-V core initialization. This way, the dispatch table can remain hidden.

It sounds good. Let's enforce the consistent schema.